backend/danswer/chat/answer.py

```diff
@@ -224,6 +226,7 @@ def _get_response(self, llm_calls: list[LLMCall]) -> AnswerStream:
         answer_handler = CitationResponseHandler(
             context_docs=search_result,
             doc_id_to_rank_map=map_document_id_order(search_result),
```

Comment:
Would create the single object with both in this function here, so above we can do processed_search_result, displayed_search_results